avocado/utils/ssh.py

```diff
@@ -155,11 +156,17 @@ def cmd(self, command):
 
         :param command: the command to execute over the SSH session
         :param command: str
+        :param command: when True run subprocess in the background
+        :param bg_processes: bool
         :returns: The command result object.
         :rtype: A :class:`CmdResult` instance.
         """
         cmd = self._ssh_cmd(self.DEFAULT_OPTIONS, ('-q', ), command)
-        return process.run(cmd, ignore_status=True)
+        sub_process = SubProcess(cmd)
```

Comment:
this line should also be moved in if condition as it only makes sense when user want to run in command  background but the only tricky part is how we terminate the command

Reply:
As the `process` module is already imported, `sub_process = process.SubProcess(cmd)` makes the code more consistent and avoid the need to do line 10 `from . process import SubProcess`.

Reply:
ok, i will chnage that.